modules/internal/OwnedObject.chpl

```diff
@@ -231,7 +231,6 @@ module OwnedObject {
   pragma "no doc"
   pragma "donor fn"
   pragma "auto copy fn"
-  pragma "erroneous autocopy"
   pragma "unsafe"
```

Comment:
It looks like this was the only thing using "erroneous autocopy", can the flag be removed from the compiler?

Reply:
The compiler uses it to generate errors for the case of no copy initializer defined for a type.

Reply:
Hmm.  I could have sworn that I searched the compiler for this flag and saw no other uses, but trying again this morning I see them.  Thanks!